src/coreclr/jit/simd.cpp

```diff
@@ -130,6 +130,29 @@ unsigned Compiler::getSIMDInitTempVarNum(var_types simdType)
     return lvaSIMDInitTempVarNum;
 }
 
+#ifdef TARGET_ARM64
+//------------------------------------------------------------------------
+// Get, and allocate if necessary, the SIMD temp used for various operations.
+// The temp is allocated as the maximum sized type of all operations required.
+//
+// Arguments:
+//    simdType - Required SIMD type
+//
+// Returns:
+//    The temp number
+//
+unsigned Compiler::getFFRegisterVarNum()
+{
+    if (lvaFfrRegister == BAD_VAR_NUM)
+    {
+        lvaFfrRegister                  = lvaGrabTempWithImplicitUse(false DEBUGARG("Save the FFR value."));
```

Comment:
I'm not sure if it matters, but this does not need to be "implicit use" anymore I think.

Reply:
yes, I had this when I was implementing this in importer. don't need it now.